services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -393,7 +382,8 @@ export const SubmissionSummary = (): React.ReactElement => {
                 ) : null}
 
                 <SubmissionTypeSummarySection
-                    submission={submission}
+                    submission={packageData}
+                    submissionName={submissionName(packageData, statePrograms)}
```

Comment:
maybe don't need to surface `submissionName` as a prop at all. looks like its a calculated value, we could calculate inside each XYZSummarySection

Reply:
It’s computed but requires the {program.id: program.nickname} mapping to compute, if we have to pass something down I’d rather it be the submissionName I think.